python/src/JunctionTreeBernsteinCopula_doc.i.in

```diff
@@ -0,0 +1,67 @@
+%feature("docstring") OTAGRUM::JunctionTreeBernsteinCopula
+"JunctionTreeBernsteinCopula distribution
+
```

Comment:
Your commit comment could be useful here:
"This class allows to build a nonparametric copula from a given sample, a bin count and a junction tree descibing the conditional independencies in the data. It is based on a Bernstein estimation of each sub-copulas of the model."

Reply:
done